ipycytoscape/cytoscape.py

```diff
@@ -811,12 +818,21 @@ class CytoscapeWidget(DOMWidget):
 
     graph = Instance(Graph, args=tuple()).tag(sync=True, **widget_serialization)
 
-    def __init__(self, **kwargs):
+    def __init__(self, graph_file=None, **kwargs):
         super(CytoscapeWidget, self).__init__(**kwargs)
 
         self.on_msg(self._handle_interaction)
         self.graph = Graph()
 
+        if isinstance(graph_file, nx.Graph):
+            self.graph.add_graph_from_networkx(graph_file)
+        if isinstance(graph_file, dict):
+            self.graph.add_graph_from_json(graph_file)
+        if isinstance(graph_file, DataFrame):
+            self.graph.add_graph_from_df(graph_file)
+        if isinstance(graph_file, str):
+            self.graph.add_graph_from_json(graph_file)
+
```

Comment:
Combine the checks that lead to `json`

```suggestion
        if isinstance(graph_file, nx.Graph):
            self.graph.add_graph_from_networkx(graph_file)
        if isinstance(graph_file, (dict, str)):
            self.graph.add_graph_from_json(graph_file)
        if isinstance(graph_file, DataFrame):
            self.graph.add_graph_from_df(graph_file)
```

Reply:
Also need a check for `neo4j`